enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -27,3 +34,72 @@ def update_course_title_for_subsidy_request_task(subsidy_type, subsidy_request_u
 
     # Use bulk_update so we don't trigger save() again
     subsidy_model.bulk_update([subsidy_request], ['course_title'])
+
+
+@shared_task(base=LoggedTaskWithRetry)
+def send_admins_email_with_new_requests_task(enterprise_customer_uuid):
+    """
+    Subsidy type agnostic task to send new-request emails to admins
+
+    Args:
+        enterprise_customer_uuid (str): enterprise customer uuid identifier
+    """
+    braze_client = BrazeApiClient()
+    lms_client = LmsApiClient()
+    enterprise_customer_data = lms_client.get_enterprise_customer_data(enterprise_customer_uuid)
+
+    contact_email = enterprise_customer_data.get('contact_email')
+    if not contact_email:
+        logger.warning(
+            f'No contact email set for {enterprise_customer_uuid}. '
+            'Not sending new subsidy request email to admin.'
+        )
+        return
+
+    config_model = apps.get_model('subsidy_request.SubsidyRequestCustomerConfiguration')
+    customer_config = config_model.objects.get(
+        enterprise_customer_uuid=enterprise_customer_uuid,
+    )
+    license_model = get_subsidy_model(SubsidyTypeChoices.LICENSE)
+    license_subsidies = license_model.objects.filter(
+        enterprise_customer_uuid=enterprise_customer_uuid,
+        state=SubsidyRequestStates.REQUESTED,
+    )
+    coupon_model = get_subsidy_model(SubsidyTypeChoices.COUPON)
+    coupon_subsidies = coupon_model.objects.filter(
+        enterprise_customer_uuid=enterprise_customer_uuid,
+        state=SubsidyRequestStates.REQUESTED,
+    )
+
+    # Go through all subsidy_requests, licenses and coupons, and gather
+    # their data into braze_trigger_properties, so we can send 1 email
+    # with all the bits we need.
+    braze_trigger_properties = OrderedDict()
```

Comment:
q: does this need to be an ordered dict?

Reply:
yeah. the request is to have results listed newest to oldest in the email, and while the query set is ordered, regular dicts are just hashes and order is not deterministic

Reply:
resolved: only the request need to be ordered, not the trigger properties dict.